crates/holochain_cascade/Cargo.toml

```diff
@@ -56,6 +56,8 @@ test_utils = [
   "holochain_state/test_utils",
 ]
 
+instrument = []
```

Comment:
These were missed with a previous PR that put `#[instrument(..` behind a feature. I haven't plumbed these together because it was already broken after the previous PR and I was just trying to fix warnings.